todoman/cli.py

```diff
@@ -331,7 +349,21 @@ def list(
     """
 
     sort = sort.split(',') if sort else None
-
+    before = None
+
+    # if start:
+    #     if 'before' in start:
+    #         before = True
+    #     else:
+    #         before = False
+    #     start = start[-10:]
+
+    if start: 
```

Comment:
I'd prefer if we pass `start` unmodified into `db.todos` and disassemble the list there. I.e. no additional `before` parameter, just passing a list.